src/main/uploader.ts

```diff
@@ -95,6 +82,35 @@ export class MinidumpUploader {
     }/api/${projectId}/minidump?sentry_key=${user}`;
   }
 
+  /**
+   * Create minidump request to dispatch to the transpoirt
+   */
+  private async _toMinidumpRequest(event: Event, minidumpPath: string): Promise<SentryElectronRequest> {
+    const envelopeHeaders = JSON.stringify({
+      event_id: event.event_id,
+      sent_at: new Date(timestampWithMs() * 1000).toISOString(),
+    });
+    const itemHeaders = JSON.stringify({
+      content_type: 'application/json',
+      type: 'event',
+    });
+
+    const stat = await statAsync(minidumpPath);
```

Comment:
You can avoid this stat, since you can simply query for `minidumpContent.length` below. That will return the binary size of the buffer. That way, you avoid differences between the FS metadata and the file contents, if any.